libs/wallet-ui/src/routes/settings/home/home.tsx

```diff
@@ -1,36 +1,35 @@
 import { Icon } from '@vegaprotocol/ui-toolkit'
 import { List } from '../../../components/list'
-import { useMemo } from 'react'
 import { Link } from 'react-router-dom'
 import { Page } from '../../../components/page'
 import { Paths } from '../..'
 
 export const SettingsHome = () => {
-  const items = useMemo(
-    () => [
-      {
-        title: 'App settings',
-        path: Paths.Settings.AppSettings,
-      },
-      // {
-      //   title: 'Networks',
-      //   path: Paths.Settings.Networks,
-      // },
-      {
-        title: 'Service',
-        path: Paths.Settings.Service,
-      },
-    ],
-    []
-  )
+  const items = [
+    {
+      title: 'App settings',
+      path: Paths.Settings.AppSettings,
+    },
+    // {
+    //   title: 'Networks',
+    //   path: Paths.Settings.Networks,
+    // },
```

Comment:
question (non-blocking): not sure what's going on with this commented out path, probably can be removed? 

```suggestion
    // {
    //   title: 'Networks',
    //   path: Paths.Settings.Networks,
    // },
```